projects/rizin/sydr-support-fuzzing.diff

```diff
@@ -0,0 +1,35 @@
+From 486fa6f7a1cddaf21baf61f6acc98c030c52ba04 Mon Sep 17 00:00:00 2001
+From: Theodor Arsenij Larionov <tlarionov@ispras.ru>
+Date: Mon, 15 Aug 2022 12:00:09 +0300
+Subject: [PATCH] Added support for sydr-fuzz
+
+---
+ binrz/meson.build | 1 +
+ meson_options.txt | 2 ++
+ 2 files changed, 3 insertions(+)
+
+diff --git a/binrz/meson.build b/binrz/meson.build
+index 407e127a31..beb42207df 100644
+--- a/binrz/meson.build
++++ b/binrz/meson.build
+@@ -10,6 +10,7 @@ else
+   subdir('rz-agent')
+   subdir('rz-diff')
+   subdir('rz-find')
++  subdir('rz-fuzz')
+   subdir('rz-sign')
+   subdir('rz-ax')
+ endif
+diff --git a/meson_options.txt b/meson_options.txt
+index 4c565ccb3c..b5a9456d7f 100644
+--- a/meson_options.txt
++++ b/meson_options.txt
+@@ -1,3 +1,5 @@
++option('fuzz_mode', type: 'combo', choices: ['afl', 'libfuzzer', 'sydr', 'none'], value: 'none', description: 'Specifies fuzzing configuration to build (if any)')
```

Comment:
Can you add .toml for libFuzzer + Sydr also?

Reply:
Added .toml for libfuzzer